letsdrill.rb

```diff
@@ -1,16 +1,21 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90
+    "A"
+  elsif integer >= 80
+    "B"
+  elsif integer >= 70
+    "C"
+  elsif integer >= 60
+    "D"
+  else
+    "F"
+  end
 end
```

Comment:
This isn't so bad, but it would be a bit nicer in a case structure with ranges. [Docs](http://ruby-doc.org/docs/keywords/1.9/Object.html#method-i-case)